calico_containers/tests/st/test_endpoint_commands.py

```diff
@@ -0,0 +1,151 @@
+from test_base import TestBase
+from docker_host import DockerHost
+
+
+class TestEndpointCommands(TestBase):
+    def test_endpoint_commands_mainline(self):
+        """
+        Run a mainline multi-host test using new endpoint commands.
+
+        Almost identical in function to the vagrant coreOS demo.
```

Comment:
I'd skip this comment and just put an overview of what it actually does.

We should be updating the coreOS demo to be inline with this test.  Need to agree that we will do that.